### ISSUE TITLE
Animation part random rate and offset

### ISSUE THREAD

**Issue by GenericcNameHere** [CapsAdmin/pac3#851]
So animation parts tend to play at random speeds (too slow, too fast, reversed etc) and seems to start on a random frame of the animation (possibly forcing owner cycle?) rather than the starting frame.

This issue usually doesn't happen on first wear of a pac but after a couple of wears will usually trigger it.

Initially I thought it was an issue exclusive to custom .mdl files with their own animations however I found out this also happens with the base hl2/gmod models or any model with animations.

I think this may be a Windows 10 issue as after testing with a few friends (Who were running Windows 8.1) by giving them a few of my pacs they didn't seem to have the issue, when testing on my machine and another Windows 10 machine on a different account the same issue occurred.

This issue can be seen by other players so it isn't just client-sided.

I have also tried both the Awesomium and Chromium branches of Gmod with the same result.

Correct rate:
https://i.gyazo.com/bdfc4c3d76dfb849bbe6292ebce1ed40.mp4

Broken rate:
https://i.gyazo.com/fe21ddb24ae81ede4c162b9ebddef087.mp4

**Comment by roboderpy** [comment]
It is probably your framerate and/or frame lag spikes

**Comment by GenericcNameHere** [comment]
> It is probably your framerate and/or frame lag spikes

No, my rig is pretty solid and like I said I tested on more than one device and even had friends test it out too lag isn't a factor here, though I know what you mean but if it was lag the animations would be choppy not what I'm having.
